guides/CUSTOM_CONTROLLERS.md

```diff
@@ -29,14 +29,14 @@ defmodule MyAppWeb.Router do
 
     get "/signup", RegistrationController, :new, as: :signup
     post "/signup", RegistrationController, :create, as: :signup
-    get "/login", SessionsController, :new, as: :login
-    post "/login", SessionsController, :create, as: :login
+    get "/login", SessionController, :new, as: :login
+    post "/login", SessionController, :create, as: :login
```

Comment:
since the session controller snippet below defines a `SessionController` module.